build/osx/entitlements.mac.plist

```diff
@@ -6,5 +6,7 @@
     <true/>
     <key>com.apple.security.device.camera</key>
     <true/>
+    <key>com.apple.security.cs.allow-jit</key>
```

Comment:
What's the reason to allow jit?
https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_allow-jit

Earlier, builds worked without the this key.

Reply:
This is required for notarization.
https://kilianvalkhof.com/2019/electron/notarizing-your-electron-application/

> One of the things Apple snuck in when adding the **notarization is the requirement of a hardened runtime** for your app, basically giving your application less rights by default.

> If you run a hardened runtime, it’s important to set at least two entitlements, “allow-unsigned-executable-memory” and **“allow-jit”**.

Reply:
Removed "allow-jit". The notarization works as well.